plugin/trino-base-jdbc/pom.xml

```
@@ -10,7 +10,7 @@
     </parent>
 
     <artifactId>trino-base-jdbc</artifactId>
-    <description>Trino - Base JDBC Connector</description>
+    <description>Trino - Base JDBC connector</description>
```

```suggestion
    <description>Trino - Base JDBC connector support</description>
```

or library .. since this is NOT actually a connector

---

connector library

---

ok